[0m[[0m[31merror[0m] [0m[0mC:\Users\yuda\Desktop\soDLA\src\main\scala\nvdla\sdp\NV_NVDLA_SDP_HLS_Y_idx_top.scala:99:28: not found: value chn_lut_out_pvld[0m
[0m[[0m[31merror[0m] [0m[0m    io.chn_lut_out_pvld := chn_lut_out_pvld(0)[0m
[0m[[0m[31merror[0m] [0m[0m                           ^[0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
